test/integration/scenarios/propagation/rpc/transactions/multisignature.js

```diff
@@ -81,10 +70,15 @@ module.exports = function multisignature(params) {
 			});
 
 			it('should confirm all transactions on all nodes', done => {
-				var blocksToWait =
-					Math.ceil(MAXIMUM / constants.maxTransactionsPerBlock) + 1;
+				// Adding two extra blocks as a safety timeframe
+				const blocksToWait =
+					Math.ceil(MAXIMUM / constants.maxTransactionsPerBlock) + 2;
```

Comment:
`MAXIMUM` what? Rename to something meaningful.

Reply:
Renamed to `numberOfTransactions` and `NUMBER_OF_TRANSACTIONS`.